solr/core/src/test/org/apache/solr/search/join/TestScoreJoinQPScore.java

```diff
@@ -346,8 +346,8 @@ public void testCacheHit() throws Exception {
               "true"),
           SolrException.ErrorCode.BAD_REQUEST);
     }
-    // this queries are not overlap, with other in this test case.
-    // however it might be better to extract this method into the separate suite
+    // these queries are not overlapping with other in this test case.
+    // however, it might be better to extract this method into the separate suite
```

Comment:
```suggestion
    // these queries are not overlapping with others in this test case.
    // however, it might be better to extract this method into a separate suite
```